libindy_vdr/include/libindy_vdr.h

```diff
@@ -481,6 +481,8 @@ ErrorCode indy_vdr_resolve(PoolHandle pool_handle,
 
 ErrorCode indy_vdr_set_cache_directory(FfiStr path);
 
+ErrorCode indy_vdr_set_ledger_txn_cache(size_t capacity, c_ulong expiry_offset_ms, FfiStr path);
```

Comment:
For all other functions we use `int64_t` or `int32_t`. Could `size_t` (`usize`) and `c_ulong` (`u64`) be switched to `i32` and `i64` respectively?

Reply:
we could, but switching to signed integers would cut the maximum capacity and expiry date in half, and it doesn't really make sense to me to support negative capacity or an expiry date that's in the past

Reply:
Signed numbers are mostly used in the FFI because (supposedly) Java handles those much better than unsigned ones.

Reply:
okay got it, I'll change to signed